src/Seq.Client.Log4Net/Client/Log4Net/LoggingEventFormatter.cs

```diff
@@ -253,5 +256,32 @@ static string Escape(string s)
 
             return s;
         }
+		
+		/// <summary>
+        /// GetValueAsLiteral attempts to transform the (string) object into a literal type prior to json serialization.
+        /// </summary>
+        /// <param name="value">The value to be transformed/parsed.</param>
+        /// <returns>A translated representation of the literal object type instead of a string.</returns>
+        static object GetValueAsLiteral(object value)
+        {
+            if (!(value is string))
+                return value;
+
+            var str = value.ToString();
+
+            long longBuffer;
```

Comment:
Since all longs also parse as decimals (and render to JSON in the same way), can we drop this one and just rely on the decimal attempt below? (Same in NLog).